src/geo/ui/fullscreen.js

```diff
@@ -14,47 +14,33 @@ cdb.ui.common.FullScreen = cdb.core.View.extend({
   className: 'cartodb-fullscreen',
 
   events: {
-
     "click a": "_toggleFullScreen"
-
   },
 
   initialize: function() {
-
     _.bindAll(this, 'render');
     _.defaults(this.options, this.default_options);
-
-    //this.model = new cdb.core.Model({
-      //allowWheelOnFullscreen: false
-    //});
-
     this._addWheelEvent();
-
   },
 
   _addWheelEvent: function() {
+    var self    = this;
+    var mapView = this.options.mapView;
 
-      var self    = this;
-      var mapView = this.options.mapView;
-
-      $(document).on('webkitfullscreenchange mozfullscreenchange fullscreenchange', function() {
-
-        if ( !document.fullscreenElement && !document.webkitFullscreenElement && !document.mozFullScreenElement && !document.msFullscreenElement) {
-          if (self.model.get("allowWheelOnFullscreen")) {
-            mapView.options.map.set("scrollwheel", false);
-          }
+    $(document).on('webkitfullscreenchange mozfullscreenchange fullscreenchange', function() {
+      if (!document.fullscreenElement && !document.webkitFullscreenElement && !document.mozFullScreenElement && !document.msFullscreenElement) {
+        if (self.model.get("allowWheelOnFullscreen")) {
+          mapView.options.map.set("scrollwheel", false);
```

Comment:
I don't think so, ```mapView.map``` should work.
https://github.com/CartoDB/cartodb.js/blob/develop/src/geo/map.js#L584
In any case this is inherited code.